scripts/ide-run-scss-lint-from-docker.sh

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env bash
-
-# This script will run scss-lint
-
-PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
-
-docker run --rm \
-  -v $PROJECT_DIR:/style-guide \
-  brainly/style-guide bash -c "scss-lint /style-guide/$1 -f JSON | sed 's/\/style-guide\///g'"
+SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source ${SCRIPT_DIR}/run-scss-lint.sh $1 -f JSON
```

Comment:
@marcinb For me it is straightforward - the script runs scss lint from docker (a call to another script is an implementation detail).